client/packages/system/src/Immunisation/VaccineCourseView/AppFooterComponent.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import {
+  Box,
+  ButtonWithIcon,
+  useTranslation,
+  AppFooterPortal,
+  LoadingButton,
+  useNotification,
+} from '@openmsupply-client/common';
+import { CloseIcon, SaveIcon } from '@common/icons';
+
+interface VaccineCoursesFooterProps {
+  isDirty: boolean;
+  cancel: () => void;
+  save: () => void;
+  isLoading: boolean;
+}
+
+export const AppFooterComponent = ({
+  isDirty,
+  save,
+  cancel,
+  isLoading,
+}: VaccineCoursesFooterProps) => {
+  const t = useTranslation('coldchain');
+  const { error, success } = useNotification();
+
+  const onSave = async () => {
+    try {
+      await save();
+      success(t('messages.vaccine-course-saved'));
+    } catch (e) {
+      error(t('error.vaccine-course-update-failed'));
```

Comment:
```suggestion
      success(t('messages.vaccine-course-saved'))();
    } catch (e) {
      error(t('error.vaccine-course-update-failed'))();
```

Need to call these for them to fire :)